core/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -892,7 +512,7 @@ public String toString() {
         }
         if (request.source() != null) {
```

Comment:
I think we should be able to simply this toString quite a bit. Shouldn't it just print out the string representation of the inner SearchRequest?